indigo_app/templates/indigo_api/task_detail.html

```diff
@@ -1,178 +1,245 @@
+              {% include 'indigo_app/actions/_action.html' with action=action this_task=True ignore_place=True %}
+            </li>
+          {% endfor %}
+          {% get_comment_list for task as comment_list %}
+          {% for comment in comment_list %}
+            <li class="activity-item card">
+              <div class="card-header">
+                {% user_profile task.created_by_user %} commented <span class="time-ago"
+                                                                        data-timestamp="{{ comment.submit_date|date:'c' }}">{{ comment.submit_date|date:"Y-m-d H:i" }}</span>
+              </div>
+              <div class="card-body">
+                {{ comment.comment | safe }}
+              </div>
+            </li>
+          {% endfor %}
+          {% if task.anchor_id %}
+            <li class="activity-item">
+              {% with task.resolve_anchor as anchor %}
+                {% if anchor.element %}
+                  Here is
+                  <a href="{% url 'document' doc_id=task.document.pk %}?toc={{ anchor.toc_entry.qualified_id|default:'' }}&anntn={{ task.annotation.id }}">{{ anchor.toc_entry.title }}</a>
+                  as it appears currently
+                  {% if not anchor.exact_match %}(<b>{{ task.anchor.id }}</b> may have been moved or
+                    removed){% endif %}
+                  <div class="sheet-outer"
+                       {% if not anchor.is_toc_element %}data-highlight="{{ task.anchor.id }}"{% endif %}>
+                    <div class="sheet-inner is-fragment">
+                      <div class="akoma-ntoso country-{{ anchor.document.country }}">{{ anchor.toc_element_html|safe }}</div>
+                    </div>
                   </div>
-                </div>
-              {% else %}
-                The content at <b>{{ anchor.toc_entry.title|default:task.anchor.id }}</b> is not available and may have been moved or removed.
-              {% endif %}
-            {% endwith %}
-          </li>
+                {% else %}
+                  The content at <b>{{ anchor.toc_entry.title|default:task.anchor.id }}</b> is not
+                  available and may have been moved or removed.
+                {% endif %}
+              {% endwith %}
+            </li>
+          {% endif %}
+        </ul>
+        {% if user.is_authenticated %}
+          {% get_comment_form for task as form %}
+          <form action="{% comment_form_target %}" method="post">
+            {% csrf_token %}
+            <textarea id="id_comment" name="comment" class="form-control" maxlength="3000" rows="10"
+                      cols="80" placeholder="Add comment" required></textarea>
+            <input type="hidden" name="honeypot" id="id_honeypot">
+            {{ form.content_type }}
+            {{ form.object_pk }}
+            {{ form.timestamp }}
+            {{ form.security_hash }}
+            <div class="form-group">
+              <input type="hidden" name="next"
```

Comment:
Minor comment: with HTML 5, unclosed tags (like `br` and `input`) don't need to be self-closed, they can simply exist without a closing tag.

Reply:
thank you for the info.